### PR TITLE
LoginPage 

### PR DESCRIPTION
**Pull request by Prag03**
- login page added
- html and css used

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,8 @@
   },
   "dependencies": {
     "@iconify/react": "^4.1.0",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.11.3",
```
Comment (codadept) [inline package.json]: Uninstall these libraries as it's not being used

File: src/Pages/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            <img src="/assets/Google.png" alt="logo" className={styles.img} />
+          </a>
+        </div>
+        <div>
+          <a href="/">
+            <img src="/assets/Facebook.png" alt="logo" className={styles.img} />
+          </a>
+        </div>
+      </div>
+    </div>
```
Comment (codadept) [inline src/Pages/Login/Login.jsx]: Check how to handle form input in react. By handle I mean, I need you to store the form input data (email, password...) in a React state such that it's two way data binding. i.e

1) When user updates the value in HTML element it get's updated in the React state
2) When we update the data in React state, it gets updated in the DOM.

[Checkout this article on Controlling Form Input (multiple)](https://ibaslogic.com/simple-guide-to-react-form/#controlled-react-form-input)

**Review by codadept — CHANGES_REQUESTED** [review]
1) Delete the Facebook.png and Google.png from `public/` and `src/Pages/Login/` because you are using the images from `public/assets/`

2) Suggestion: If you need to use images then always put the images in public directory.

Suppose you image is in `/public/assets/background.png`

and to use it you have to write the source of the image as

```html
<img src="/assets/background.png" alt="background"/>
```

```css
.class-name{
    background-image: url("/assets/background.png");
}
```

because vite servers the public directory as root (/). [Checkout](https://vitejs.dev/guide/assets.html#the-public-directory)

3) Complete the styling of the page

**Comment by Prag03** [comment]
Changes done.
New pages added.
RecoveryMail URL : http://localhost:3000/forgotpassword
RecoveryCode URL : http://localhost:3000/sendotp
ResetPassword URL : http://localhost:3000/verified

**Comment by codadept** [comment]
@Prag03 The numbers are not visible. Maybe a padding issue. Fix it

image

File: src/Pages/RecoveryCode/RecoveryCode.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+              onChange={(e) => handleChange(e, index)}
+              onKeyUp={(e) => handleKeyUp(e, index)}
+              ref={(ref) => {
+                refs.current[index] = ref;
+              }}
+            />
+          ))}
+          <a className={styles.resendotp} href="/">
+            Resend OTP
+          </a>
```
Comment (codadept) [inline src/Pages/RecoveryCode/RecoveryCode.jsx]: For internal links use

`Link` component from `react-router-dom`

**Review by codadept — CHANGES_REQUESTED** [review]

**Comment by Prag03** [comment]
Fixed bugs.
Changes done.

**Review by codadept — APPROVED** [review]
Good job!

**Pull request merged**